pkg/ery/cmd/cmd.go

```diff
@@ -71,7 +75,8 @@ func setupLogger(verbose bool) {
 
 func runCommand(c di.AppComponent, name string, args []string) error {
 	log := zap.L().Named("exec")
-	eg, ctx := errgroup.WithContext(context.Background())
+	cctx, cancel := context.WithCancel(context.Background())
```

Comment:
**[govet]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
the cancel function is not used on all paths (possible context leak)